tests/cxx/badinc.cc

```diff
@@ -1094,8 +1094,6 @@ int main() {
   // IWYU: I1_PtrDereferenceClass needs a declaration
   I1_PtrDereferenceClass* local_i1_ptrdereference_class = 0;
   int x;
-  // IWYU: va_list is...*<stdarg.h>
```

Comment:
Please squash this commit into `Add more mappings for 'va_list' `.

Reply:
Squashed.

Reply:
Oops, I didn't see what this did the first time around. I think we should keep the `va_list` usage around, and instead check that it's attributed to `<stdio.h>`.